ggml-cuda.h

```diff
@@ -39,6 +40,8 @@ void dequantize_row_q5_0_cuda(const void * vx, float * y, int k, cudaStream_t st
 void dequantize_row_q5_1_cuda(const void * vx, float * y, int k, cudaStream_t stream);
 void dequantize_row_q8_0_cuda(const void * vx, float * y, int k, cudaStream_t stream);
 
+cudaError_t ggml_cuda_cpy_tensor2D(void * dst, const struct ggml_tensor * src, uint64_t i3, uint64_t i2, cudaStream_t stream);
```

Comment:
Maybe the function name should make it clear that this is only for HostToDevice copies.

Reply:
It's kind of a long name but what would you prefer:
```
ggml_cuda_cpy_tensor2d_h2d
ggml_cuda_memcpy_h2d_2d
ggml_cuda_memcpy_to_device_2d
```

I feel 2D should be mentioned because it only copies a partial tensor.

Reply:
How about something like `ggml_cuda_h2d_tensor_2d`? The h2d already implies that it is a copy.